example_cycles/wet_propulsor.py

```diff
@@ -0,0 +1,192 @@
+import openmdao.api as om
+
+import pycycle.api as pyc
+
+
+class WetPropulsor(om.Group):
+
+    def initialize(self):
+        self.options.declare('design', types=bool, default=True)
+        self.options.declare('WAR', default=.001, desc='water to air ratio by mass (specific humidity)')
+
+    def setup(self):
+
+        thermo_spec = pyc.species_data.wet_air #special species library is called that allows for using initial compositions that include both H and C
+        design = self.options['design']
+        WAR = self.options['WAR']
+
+        self.add_subsystem('fc', pyc.FlightConditions(thermo_data=thermo_spec,
```

Comment:
you made WAR and option. It should be a new input to FC though... We should differentiate wrt it